Makefile

```diff
@@ -38,9 +38,9 @@ FFTW_LIBRARY       =
 
 # GNU Scientific library flags and paths.  If paths are empty, tries standard default linux paths.
 #--------------------------------------------------------------------------------------------------
-GSL                = 0
+GSL                = 1
 GSL_INCLUDE        =
-GSL_LIBRARY        =
+GSL_LIBRARY        = 
```

Comment:
Can you point me to the commit or apply the fix yourself? I can't spot it in icsilcc